apps/test/unit/templates/curriculumCatalog/curriculumCatalogExpandedCardTest.jsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import {expect} from '../../../util/reconfiguredChai';
+import ExpandedCurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/ExpandedCurriculumCatalogCard';
+import {translatedAvailableResources} from '@cdo/apps/templates/teacherDashboard/CourseOfferingHelpers';
+import {Provider} from 'react-redux';
+import {
+  getStore,
+  registerReducers,
+  restoreRedux,
+  stubRedux,
+} from '@cdo/apps/redux';
+import teacherSections from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import sinon from 'sinon';
+describe('CurriculumCatalogExpandedCard', () => {
+  let defaultProps;
+  let store;
+
+  const renderCurriculumExpandedCard = (props = defaultProps) =>
+    render(
+      <Provider store={store}>
+        <ExpandedCurriculumCatalogCard {...props} />
+      </Provider>
+    );
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers({teacherSections});
+    store = getStore();
+    defaultProps = {
+      courseDisplayName: 'AI for Oceans',
+      duration: 'quarter',
+      gradeRange: 'Grades: 3-12',
+      subjectsAndTopics: ['Artificial Intelligence', 'Data'],
+      deviceCompatibility:
+        '{"computer":"ideal","chromebook":"ideal","tablet":"ideal","mobile":"ideal","no_device":"incompatible"}',
```

Comment:
For the checks later on the device compatibility, could you make one of them "not recommended" please so we have all 3 possible states covered in the default props?